README.md

```diff
@@ -466,16 +471,20 @@ On the client side, to initiate the request:
 
 On the server side, here is an example response header:
 ```text
-Shared-Storage-Write: clear, set;key="hello";value="world";ignore_if_present, append;key="good";value="bye", delete;key="hello", set;key="all";value="done"
+Shared-Storage-Write: clear, set;key="hello";value="world";ignore_if_present, append;key="good";value="bye", delete;key="hello", set;key="all";value="done", options;with_lock="report-lock"
 ```
 
-Sending the above response header would be equivalent to making the following calls in the following order on the client side, from either the document or a worklet:
+Sending the above response header would be equivalent to making the following call on the client side, from either the document or a worklet:
 ```js
-sharedStorage.clear();
-sharedStorage.set("hello", "world", {ignoreIfPresent: true});
-sharedStorage.append("good", "bye");
-sharedStorage.delete("hello");
-sharedStorage.set("all", "done");
+
+sharedStorage.batchUpdate([
+  new SharedStorageClearMethod(),
+  new SharedStorageSetMethod("hello", "world", {ignoreIfPresent: true}),
+  new SharedStorageAppendMethod("good", "bye"),
```

Comment:
Suggest adding a lock to one of these ops to show that it can be done.

Reply:
Done.